docs/Displaying a User Interface/Main Menu/index.md

```diff
@@ -1,4 +1,8 @@
 # Main Menu
+@![javscript]
+The SDL JavaScript Suite currently does not support the MenuManager. This will be addressed in a future release.
+!@
```

Comment:
This is not appearing on the main menu build that I can see. You can include the "Using RPCs" section at the bottom.